### PR TITLE
Added a note that this Boost build method works using the Windows Com…

### PR DESCRIPTION
**Pull request by Vultraz**
…mand Prompt

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -37,3 +37,6 @@ dist\bin\bcp.exe algorithm asio assign bimap container date_time dynamic_bitset
 ```
 
 Replace the outdated files in 'external/lib' with those from 'boost_.../stage/lib' and those in 'external/include/boost' with  the ones in '_include/boost'.
+
+### Note for non-Visual Studio users
+Boost may also be build using this method in the regular Windows Command Prompt, though a different toolset needs to be specified (such as `--toolset=gcc`) and `.\b2` must be invoked as an executable (`b2.exe`).
```
Comment (aquileia) [inline README.md:42]: 'be build' -> 'be built'
I know it's nitpicky, but as the Readme is te only file a user will read, we may as well be ;)

**Comment by Vultraz** [comment]
Geez, has this been sitting here for over 2 years simply because I didn't fix a typo? :|

**Comment by Wedge009** [comment]
I don't know the history of this repository, and I'm not sure if @aquileia is even still around... I know it's only a Windows library repository for convenience, but just wondering if it's worth forking it into the Wesnoth GitHub account, if you're concerned about being able to update it in a timely manner. Not sure when or why but it seems only myself - and maybe @jyrkive - were given push access.

**Comment by gfgtdf** [comment]
I also have push access, but ye if aquileia is gone forever we should probably fork it to wesnoth.

**Comment by Vultraz** [comment]
Honestly, a git repo isn't the best place for something like this since it keeps all the history.

**Comment by Wedge009** [comment]
True - my work enforces a no-large-binaries rule for our git repositories. We have a lot of large binaries (music, images, etc) for Wesnoth itself. Isn't there Git LFS for that sort of thing? I don't think I've dealt with it much before.

**Comment by jyrkive** [comment]
I don't have push access here.

Regarding Git LFS, I think it requires the files to be stored somewhere outside the Git repo itself (and thus costs money to use, unless we can host the files on our own infrastructure).

And yes, I think it would be best to move this repository under control of the wesnoth project.

**Comment by CelticMinstrel** [comment]
For the record, I also have push access. However, I don't think a git repository is the best way to handle this.

**Pull request merged**